### ISSUE TITLE
Integration best practices doc is missing in 4.1.0 docs

### ISSUE THREAD

**Issue by chanikag** [wso2/docs-apim#5638]
**Description:**
Integration best practices doc [1] is missing in 4.1.0 docs

[1] https://ei.docs.wso2.com/en/latest/micro-integrator/references/best-Practices/#working-with-endpoints

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by Mariangela** [comment]
Hi @chanikag , 

Thank you for reporting this issue.

- Do you suggest that we add the entire content in [1] as it is into the APIM docs? 
- Is it correct to understand that this issue is applicable to the 4.0.0, 4.1.0 and master branches

[1] https://ei.docs.wso2.com/en/latest/micro-integrator/references/best-Practices

**Comment by chanikag** [comment]
Yes this doc is applicable to 4.0.0, 4.1.0 and master branches. (Not sure the content is spread over multiple pages). Let's bring it to the 4.0.0, 4.1.0 and master branches as this content is informative and usually referred.

**Comment by Mariangela** [comment]
@samgnaniah , as you were involved in moving the EI docs to API-M, would you happen to know if this content spread over multiple pages. Please work on this Git issue when time permits.
